### DISCUSSION TITLE
MacOs notification without options

### DISCUSSION THREAD

**Discussion by kristiannotari** [tom-james-watson/breaktimer-app#226] [Q&A] [answered]
Hello, with macos I switched from whole screen popups to notifications. I was expecting the "start break" / "snooze" options to be there as notifications dropdown options and for breaks to automatically start after a delay without user action, but that doesn't seem the case as there's no dropdown with options inside the notification, which simply states the message for a break.

Is that a bug on my end or is that not a currently supported feature at all?

![image](https://github.com/tom-james-watson/breaktimer-app/assets/17885006/09a4a5a5-a4f0-4954-a8d6-4425d178f57f)

**Comment by tom-james-watson** [comment]
That is the expected functionality with notifications only. How would a break "start" if doing notifications only?

**Reply by kristiannotari** [reply]
after a small delay as the one with the screen overlay. The thing is, during that delay I would be able to still use my mouse/input to my pc instead of getting a little "freeze" and no responsiveness to the system. During that delay I could also use the dropdown menu from the notification (which is not there unfortunately) to skip or snooze the break

**Reply by tom-james-watson** [reply]
So one native notification that says "break about to start..." with notification options for skip/snooze, followed a few seconds later by a second native notification that says "time for a break..." etc.?

To me having two separate notifications like this seems pointless, to be totally honest.

**Reply by kristiannotari** [reply]
why two? The whole screen overlay doesn't "disappear" and then appear again with two different messages. It stays there, giving you the ability to snooze or skip. The same for the notification, notifying you it's time to start a break. It could well be that it start the whole screen "break timer" after the initial delay with no skip or snooze, or maybe not, or maybe that should be customizable.

**Reply by tom-james-watson** [reply, marked as answer]
I don't see it, I'm afraid. How it works at the moment is the intended UX.

**Reply by kristiannotari** [reply]
the whole point of this is to be able to be notified AND being able to skip or snooze, without losing focus to what's one dealing with with its computer, maybe it's writing in a form and the responsivenss get interrupted a few milliseconds before without prior warning (rightly so). This is somehow disturbing when it keeps happening while you're almost there to finish something important before your break, so having the notification popup instead of the whole screen focus-breaking alert seems like a nice tradeoff

**Reply by kristiannotari** [reply]
customizable obviously, not something that should be enforced on everyone
